community/kernel/src/main/java/org/neo4j/kernel/AvailabilityGuard.java

```diff
@@ -281,6 +281,7 @@ private Availability availability( long millis )
             return availability;
         }
 
+        log.warn( "Database is unavailable awaiting for it to be available again for " + millis + "ms" );
```

Comment:
I can change it no big deal. Why do you think `info` is more appropriate than `warn`?